cactusbot/commands/magic/command.py

```diff
@@ -91,3 +117,30 @@ class Meta(Command):
         response = await self.api.command.update_count(command, action_string)
         if response.status == 200:
             return "Count updated."
+
+    @Command.command(role="moderator")
+    async def permission(self, command: r'?command', role=None):
```

Comment:
We did, but not using that for permissions. That becomes a problem of having to readd the command to change both, and right now I would prefer if the two were separated.